### PR TITLE
Docode morse code

### PR DESCRIPTION
**Pull request by gedewon**
## PR checklist;
- [x] Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. decode_char(".-") returns "A").

- [x] Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

- [x] Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.

  decode("-- -.--   -. .- -- .")
returns "MY NAME").

Reuse methods whenever possible. Make sure that your code is DRY.

### CODE REVIEW

File: index.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  decoded_string.join
+end
+
+def decode(input)
+  array_of_words = input.split('   ')
+  decoded_string = []
+  array_of_words.each do |element|
+    decoded_string.push(decode_word(element))
+  end
+  puts decoded_string.join(' ')
```
Comment (TedLivist) [inline index.rb]: - [x] You are required to return the decoded message, not print it. As it is this returns `nil`. Kindly remove the `puts` keyword here.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Gedewon and @kinginthenorthcodez,

Good job so far!
✔️ Reuse of methods
✔️ Decode of character and word
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.rb (hunk 1 above)
Comment (gedewon) [reply index.rb]: Great !! Nice catch

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Gedewon     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**